homeassistant/components/myuplink/sensor.py

```diff
@@ -86,13 +101,41 @@
         state_class=SensorStateClass.MEASUREMENT,
         native_unit_of_measurement=UnitOfVolumeFlowRate.CUBIC_METERS_PER_HOUR,
     ),
+    "min": SensorEntityDescription(
+        key="minutes",
+        device_class=SensorDeviceClass.DURATION,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfTime.MINUTES,
+        suggested_display_precision=0,
+    ),
+    "Pa": SensorEntityDescription(
+        key="pressure_pa",
+        device_class=SensorDeviceClass.PRESSURE,
+        state_class=SensorStateClass.MEASUREMENT,
+        native_unit_of_measurement=UnitOfPressure.PA,
+        suggested_display_precision=0,
+    ),
```

Comment:
There's already a pressure sensor reporting in bar, is this the same data, just with a different unit?

Reply:
Yes, I made it this way so that the specific equipment pressure sensor will display the value with same unit as in the myUplink App.

Reply:
Just to double confirm, we create multiple sensors in Home Assistant for the same data with different units?
What's the point of that, why not just a single pressure sensor which the user can customize to the preferred unit?

Same question for the duration sensors.

Reply:
OK, and the user can differentiate between, for example multiple pressure sensors, because the name of the entity is set to `device_point.parameter_name` which is descriptive enough to make sense?